### PR TITLE
Issue #652, PT-10: As a clinician, I see growth charts generated using the latest WHO data

### PR DESCRIPTION
**Pull request by danielpgross**
Pending QA runs in PT-17, PT-19

### CODE REVIEW

**Comment by danielpgross** [comment]
@antonkatz @itaiGershtansky for the review please!

File: components/patient-measurements/api/src/main/resources/org/phenotips/measurements/internal/hc.csv [hunk 1]
```diff
@@ -1,3743 +1,91 @@
...
-2,732,1,47.1891,0.02957
-2,733,1,47.1937,0.02957
-2,734,1,47.1983,0.02957
-2,735,1,47.2029,0.02957
-2,736,1,47.2075,0.02957
-2,737,1,47.2121,0.02957
-2,738,1,47.2167,0.02956
-2,739,1,47.2213,0.02956
-2,740,1,47.2258,0.02956
-2,741,1,47.2
```
Comment (antonkatz) [inline components/patient-measurements/api/src/main/resources/org/phenotips/measurements/internal/hc.csv]: This one. This one is definitely wrong.

**Comment by antonkatz** [comment]
+1

File: components/patient-measurements/api/src/main/resources/org/phenotips/measurements/internal/hc.csv (hunk 1 above)
Comment (danielpgross) [inline components/patient-measurements/api/src/main/resources/org/phenotips/measurements/internal/hc.csv]: :laughing:

File: components/patient-measurements/api/src/main/resources/measurementsChartsConfigurations.properties [hunk 2]
```diff
@@ -125,12 +134,16 @@ charts.bmi.configurations.03.upperValueLimit=23.8
 charts.bmi.configurations.03.chartTitle=Body mass index, birth to 36 months
 charts.bmi.configurations.03.topLabel=Age (months)
 charts.bmi.configurations.03.leftLabel=Body mass index
+charts.bmi.configurations.03.chartSource=The WHO Growth Charts for Canada
+charts.bmi.configurations.03.chartSourceLink=http://www.whogrowthcharts.ca/
```
Comment (sdumitriu) [inline components/patient-measurements/api/src/main/resources/measurementsChartsConfigurations.properties]: These are still disabled (see the commented out above), should they be enabled?

Comment (danielpgross) [inline components/patient-measurements/api/src/main/resources/measurementsChartsConfigurations.properties]: Oops, yes they were meant to be enabled. Will fix.

**Pull request merged**